tabpy-server/tabpy_server/handlers/base_handler.py

```diff
@@ -1,30 +1,143 @@
 import base64
 import binascii
 import concurrent
-import tornado.web
 import json
 import logging
-
+import tornado.web
 from tabpy_server.app.SettingsParameters import SettingsParameters
 from tabpy_server.handlers.util import hash_password
+import uuid
+
 
-logger = logging.getLogger(__name__)
 STAGING_THREAD = concurrent.futures.ThreadPoolExecutor(max_workers=3)
 
 
-class BaseHandler(tornado.web.RequestHandler):
-    KEYS_TO_SANITIZE = ("api key", "api_key", "admin key", "admin_key")
+class ContextLoggerWrapper(object):
```

Comment:
Declaring classes like this is also an artifact from new / old style classes in Python2. Since you don't support that any more would give your codebase a more modern look if you just did `class ContextLoggerWrapper` (no functionality will change)